src/registrar/templates/application_withdraw_confirmation.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'base.html' %}
+
+{% block title %}Withdraw request for {{ domainapplication.requested_domain.name }}{% endblock %}
+{% load static url_helpers %}
+
+{% block content %}
+<div class="grid-col desktop:grid-offset-2 desktop:grid-col-8">
+ 
+
+<p><h1>Withdraw request for {{ domainapplication.requested_domain.name }}?</h1></p>
+
+If you withdraw your request we won't review it. Once you withdraw your request you'll be able to edit it or completely remove it. 
+
+<p><a href="{% url 'application-withdrawn' domainapplication.id %}" class="usa-button withdraw">Withdraw request</button>  
```

Comment:
The space the two buttons is a little tight, and I think also causing the no-CSS issue Seamus noted.  e could either have them as just anchor tags, or as individual `p` tags, that are then grouped together and styled with `flex` or `inline-blocks`

Reply:
i found a problem, i fixed it.